docs/protocol.md

```diff
@@ -182,15 +182,35 @@ the server using the streamable transport protocol.
 
 #### Stateless Mode
 
-<!-- TODO -->
-
-#### Sessionless mode
-
-<!-- TODO -->
+The streamable server supports a _stateless mode_ by setting
+[`StreamableHTTPOptions.Stateless`](https://pkg.go.dev/github.com/modelcontextprotocol/go-sdk/mcp#StreamableHTTPOptions.Stateless),
+which is where the server does not perform any validation of the session id,
```

Comment:
end prev sentence with period

then "In stateless mode, the server..."

Reply:
also s/id/ID/

Reply:
Done (x2).